api/eapp.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDKuhoaORIs+cfQ
+jjMKKoWrxvF2wfHqjT0Y1OHL0vYXyd170pCGw4AnVYtE/nszc3LNxSuqkxcjkoU7
+AWzSXLI5yI7qESiqTkHRG0l1hnUOnFkQRzkjT3sgFsd3Dw38LLf7x0oEDjN3Q2yZ
+n30pTzCKJnrImP0NpbBSM4SlTOJjG5KPtOIMS2k4RU80P0OqPaxZc3v2qzZWHxpG
+OrKcpbLDxr1/FK+mWm92lVVQ45H4No9LRkAjaFfGZR9i2zykKLz7CT50YzCcEiki
+DscWR5SPN4B+J/+WNV1KiraqqbK2k+LlY9L8iVTUwxLYI6r/gYe50oxFzTR+8Xzw
+WSWj8e1HAgMBAAECggEAe6ouxhROXBU6qKQ1ylvKEtFW9qS0l+n0fTNPUfCsojJH
+44K9GxgjEtuCC2JZcTdZruA6FOGJWe/FSgtCgMh/B/SqCWrSLaFzUMWeeAy/V1xU
+kDOCTWHsr9R1dNFALLZyvSoCopqvFm+Qzd2Wkh52c6+pcriwlSSffGVk4xkVeHKB
+Xc3viwqlHMXXbzdFC3YrBZbZy7A47IpeBmW1QnZGO+uJ76G5tbna1bOoAzCRWUu7
+gwD8KzB16iCnKItD9X0zcUjcbaRzKRnU4SsZ1hmmt2kQKXxY6QsQhgPO+dtfDJNM
+LQ9AbAdBHRCclc4NBRAJb8+MmnQ04dWK5/rskENYAQKBgQDpdiMfXm6GkUmx7h88
+0MWZCwx3L/1+VrlwaALqu7b3ZdrKCjr8CV64ELM4hmHKFdcHRzrMWHSVpFrGHLy5
+HWIrpt8ygkW5PjrBGuB5tzELYggiLyO7/dPATuLM2imqZcVTrV9P9jJXARRGNa3/
+5HPLQx8LyHi1a9QCVUw8UtbHRwKBgQDeTGEfnRWbVWVZFRuoayDrnmipchM8e0wI
+dLpdFGGlIfKf2DWpEFBAieS9X4aoJjfMdvjSyBEKfS880mEU4a94euDEP6s/d3mL
+LU68bSqPkr8NqMCaFcm8vNKKqQbO2c01u9TVooLPVdRqd0h0SM50EQHGojwabIFu
+4+kp2AqqAQKBgHsxyvDnrN0317Kb3EIETPEoomhWm6bKbsqBKGDjs3YzelO5OEz9
+SM1KirheOUYhZjnB/aYNW2AHvZrjWTc56S7tvMPzeIV68BgsbZ3s7GV8UpxvUaUH
+GXFWb1n4GZqAX7N7Plt8qC2/NQqjkvEAhZYSElTucbwSbWrZxgPjFtFLAoGBANHZ
+kpoQZTRE8i6dMYXU07f5y12iNayTzFvzLXbu5ZrIqEzjAwz9PHUgd1GaGvkjUutu
+qEf627NmJ7Cn/UOEJWx7uzEn1ZZNR+ztsXVJLLNqWXcghekvhkjoi2ygUa/1hbYj
+AbFG/2uQ37w9rKUxkcuZvm5v+oMLSUiQWCzLKI4BAoGAZdWHWfgbGWdQeR8A7ld9
+CfMHlnfdXQRCxbXiHLewQZJpO6CcmhcV394SX3YFAYL8Y1FNpYMWPyCMHWm16IIf
+ISJt8kLi0mmwFMr5Kt/LAfzWr1THSuhBnyyRapZ7rFrYZ/US6Yl4CD8EPHGIIoUm
+2BFHI04rA8WhxdrjMVjTALg=
+-----END PRIVATE KEY-----
```

Comment:
Not totally sure about the context here, or if this was committed accidentially, but either way it's probably best not to have keys committed to version control, even if they are just for testing. Cloud.gov has services for credentials, these should be added there.

Reply:
If our unit tests require a test certificate as this where would you recommend storing them? Or maybe generate them on demand?

Definitely open to suggestions!

Reply:
Ah, gotcha. Yeah, I would recommend generating them on the fly in that case. Don't want to normalize seeing private keys in plain text. For the future I would recommend you & team  installing 18F's git-seekrets to prevent leaking sensitive info, available [here](https://github.com/18F/git-seekret). It works well, hooks into git's commit command and scans the commit for various patterns.

Reply:
Sweet! Added it in the backlog @ https://github.com/truetandem/e-QIP-prototype/issues/2253